CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_ActivityChainSpawner.uc

```diff
@@ -26,8 +26,11 @@ var const config array<int> WorkRequiredForSpawn;
 var const config array<int> WorkRequiredForSpawnVariance;
 
 // These 2 control the interval in which the counter-DE ops will pop
-var const config int MinCounterDarkEventDay;
-var const config int MaxCounterDarkEventDay;
+var const config int MinDarkEventWaitDays;
+var const config int MaxDarkEventWaitDays;
+
+var const config int MinGenericWaitDays;
+var const config int MaxGenericWaitDays;
```

Comment:
Since this is no longer used in this class, please move them to wherever they are used (`X2StrategyElement_DefaultActivities`)